test

```diff
@@ -1 +1,2 @@
 test
+4502812353466827
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Regex' was found.
Severity: Medium

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

 <!-- ab277f94-3666-437a-8322-8838d1b4c15c -->

Reply:
#cycode_secret_ignore_everywhere

Reply:
:-1: This user is not permitted to perform actions.